DEV_ENVIRONMENT.md

```diff
@@ -2,12 +2,16 @@
 
 1. Make sure you have `node` installed with a version at _least_ 10.0.0 and `yarn` with a version
    of at least 1.10.0. We recommend using `nvm` to manage your node versions.
-2. Run `yarn global add gulp` to install gulp.
-3. Fork the `angular/components` repo on GitHub.
-4. Clone your fork to your machine with `git clone`.
+2. Material uses Bazel which requires certain Bash and UNIX tools.
```

Comment:
```suggestion
2. angular/components uses Bazel which requires certain Bash and UNIX tools.
```

Reply:
Should it be changed further down aswell?

Reply:
Ah yes, I missed then when renaming the repo